src/components/Tokens/TokenDetails/PriceChart.tsx

```diff
@@ -382,7 +382,7 @@ export function PriceChart({ width, height, prices }: PriceChartProps) {
               key={DISPLAYS[time]}
               active={timePeriod === time}
               onClick={() => {
-                setTimePeriod(time)
+                startTransition(() => setTimePeriod(time))
```

Comment:
Why do you do this? Don't you _want_ Suspense to kick in?

Reply:
We want the current chart line to remain and then re-render once the new line is ready, rather than flash a loading state. Eventually the intention is to animate the first line into the second line.

Reply:
You should add a comment to make this clear, just something simple like your first sentence here.